content/_data/sidebar.json

```diff
@@ -37,7 +37,7 @@
           "slug": "end-to-end-testing",
```

Comment:
The title prop here can be "E2E Testing" as well so the nav reflects this change too.

Reply:
Talked about this with Brian, I feel that showing users the acronym in the top-level nav presumes too much that they already know what it means. I think users shouldn't need to be educated before understanding our navigation, does that make sense? Anyway, going to mark this as resolved for now and we can always revisit later.

Reply:
Got it.